aptos-move/framework/aptos-stdlib/doc/ed25519.md

```diff
@@ -170,32 +170,32 @@ A purported Ed25519 signature that can be verified via <code>signature_verify_st
 ## Constants
```

Comment:
This file too.

Reply:
They probably got regenerated when you ran `cargo test` in `aptos-move/framework/`. Make sure you are up to date with main.